### PR TITLE
Add link to includeEnvironmentVariables in config.ts

### PR DESCRIPTION
**Pull request by jtoar**
This PR adds a comment that points people to webpack.common.js for web's includeEnvironmentVariables. Via https://github.com/redwoodjs/redwoodjs.com/pull/173#issuecomment-644537696

### CODE REVIEW

File: packages/internal/src/config.ts [hunk 1]
```diff
@@ -39,6 +39,7 @@ export interface Config {
   }
 }
 
+// Note that web's includeEnvironmentVariables is handled in https://github.com/redwoodjs/redwood/blob/d51ade08118c17459cebcdb496197ea52485364a/packages/core/config/webpack.common.js#L19
```
Comment (thedavidprice) [inline packages/internal/src/config.ts:42]:
```suggestion
// Note that web's includeEnvironmentVariables is handled in `webpack.common.js`
// https://github.com/redwoodjs/redwood/blob/d51ade08118c17459cebcdb496197ea52485364a/packages/core/config/webpack.common.js#L19
```

Just splitting into two lines.

**Comment by thedavidprice** [comment]
@peterp Looping you in on this as it begs a question --> handling the Env Vars this way was done _prior_ to you reworking `config.ts`. Should it be added to the types and default as well?

**Comment by thedavidprice** [comment]
🚀

**Comment by jtoar** [comment]
@peterp I think `includeEnvironmentVariables` is an array of strings, so is it as simple as `string[]`? Like:

```diff
interface BrowserTargetConfig {
  name?: string
  host: string
  port: number
  path: string
  target: TargetEnum.BROWSER
  // TODO: apiProxyHost: string
  apiProxyPort: number
  apiProxyPath: string
+ includeEnvironmentVariables: string[]
}
```

**Comment by thedavidprice** [comment]
@peterp The impetus here is that you've now updated `config.ts` to represent the master `redwood.toml` config + defaults. However, it's missing the `includeEnvironmentVariables` config option and default, which is set via Webpack currently.

Thus Dom's suggestion above. Does that make sense or are we just confusing things?

**Pull request merged**